### ISSUE TITLE
Detect assignment in test condition

### ISSUE THREAD

**Issue by pshem** [donhui/sonar-mybatis#52]
It would be nice if the following was detected and flagged as always true:
```xml
<if test="potato = 'tomato'">
```

The intended code was:

```xml
<if test="potato == 'tomato'">
```

Assignments inside a test statement are unlikely to be intentional. The check could be implemented by looking for `=` signs inside a `test` statement without an adjacent `=`, `>`, `<` or `!`
